static/js/portico/signup.js

```diff
@@ -60,4 +60,12 @@ $(function () {
         errorClass: "text-error",
         wrapper: "div",
     });
+
+    $(".register-page #email, .login-page-container #id_username").on('focusout keydown', function (e) {
+        // check if it is the "focusout" or if it is a keydown, then check if
+        // the keycode was the one for "enter" (13).
+        if (e.type === "focusout" || e.which === 13) {
```

Comment:
Maybe `if (e.type === "focusout" || (e.type === "keydown" && e.which === 13)) {` is more future-proof.

Reply:
Can you elaborate on why you think so? I think this is probably a case of programming for impossible cases. The only two events that can trigger an `e.which` of `13` are `keyup` and `keydown` and arguably if someone changed the event to keyup for whatever reason, this one would still work whereas by over-specifying it would stop working.

Reply:
Yeah, it's really more for readability, but it's pretty minor.   I'll just merge, this code is most likely to break through our breaking the selectors, not something else, anyway.